python-frontend/src/test/java/org/sonar/python/tree/PythonTreeMakerMatchStatementTest.java

```diff
@@ -171,4 +174,93 @@ public void capture_pattern() {
     assertThat(capturePattern.name().name()).isEqualTo("x");
     assertThat(capturePattern.children()).containsExactly(capturePattern.name());
   }
+
+  @Test
+  public void sequence_pattern() {
```

Comment:
Can we find a way to make those tests shorter? For example, do we get more confidence from our tests by checking the result of the `children` method for each of the following cases?

Reply:
Indeed, I will simplify those tests and I will take the method you wrote inside "or patterns" PR that factorizes pattern creation  :) 
I also agree that we don't really need to check children as we already check that all tokens are there.